examples/eventlisteners/gitlab-eventlistener-interceptor.yaml

```diff
@@ -12,7 +12,6 @@ spec:
             secretRef:
               secretName: foo
               secretKey: bar
-              namespace: baz
```

Comment:
Do we also need to remove this `namespace: baz` line from [`eventlistener-multiinterceptor.yaml`](https://github.com/tektoncd/triggers/blob/master/examples/eventlisteners/eventlistener-multiinterceptor.yaml#L15)?